hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/DFSConfigKeysLegacy.java

```diff
@@ -69,9 +69,13 @@ private DFSConfigKeysLegacy() {
   public static final String DFS_DATANODE_KERBEROS_PRINCIPAL_KEY =
       "dfs.datanode.kerberos.principal";
 
+  @Deprecated
```

Comment:
We need to add the pair of old/new keys here:

https://github.com/apache/ozone/blob/d6cbb15f7a29221fcbd8f2c3bc9f5073c12c12d7/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java#L297-L306

Reply:
Thanks for the reminder, let me add it.